modules/attribution-document/attribution-document-core/src/main/java/org/eclipse/sw360/antenna/attribution/document/utils/TemplateLoaderUtil.java

```diff
@@ -32,9 +33,57 @@ private TemplateLoaderUtil() {
     }
 
     public static Templates load(String key) {
-        TemplateBundle bundle = getLoader(key);
+        return load(getLoader(key));
+    }
+
+    public static Templates load(File cover, File copyright, File content, File back) {
+        return load(new TemplateBundle() {
+            @Override
+            public String key() {
+                return "";
```

Comment:
Would it make sense to derive an "artificial" key from the files names? I'm not sure how the return value of this function is used, if at all. Maybe it's just fine to always return a empty string here.

Reply:
Actually it's fine to return only an empty string in this case, because the key is only used when loading the template via the ServiceLoader.